opensearch_py_ml/ml_commons/rag_pipeline/rag/.gitignore

```diff
@@ -0,0 +1,12 @@
+# Ignore data and ingestion directories
+ml_commons/rag_pipeline/data/
+ml_commons/rag_pipeline/ingestion/
+ml_commons/rag_pipeline/rag/config.ini
+# Ignore virtual environment
+.venv/
+# Or, specify the full path
+/Users/hmumtazz/.cursor-tutor/opensearch-py-ml/.venv/
```

Comment:
Don't add personal information to code base

Reply:
Something like this 
https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/actions.py#L1-L25

Reply:
Done, taken care of